jerry-main/main-unix.c

```diff
@@ -107,6 +107,21 @@ assert_handler (const jerry_value_t func_obj_val __attribute__((unused)), /**< f
   }
 } /* assert_handler */
 
+/**
+ * Provide the '__gc' implementation for the engine.
```

Comment:
_the_ '__gc' implementation? Is it mandated somewhere? Or is it introduced into the system for the sake of the fixed issue? Will it be useful elsewhere, too? Do other engines also expose their gc? If yes, also under this name?